docs/design/iam_containerized.md

```diff
@@ -0,0 +1,272 @@
+# Containerized Noobaa IAM
+
+## Glossary
+**Access keys** = a pair of access key ID (in short: access key) and secret access key (in short: secret key)  
+**ARN** = Amazon Resource Name  
+**CRUD** = Create, Read, Update, Delete  
+**IAM** =  Identity and Access Management  
+
+## Goal
+Ability to operate NooBaa accounts for Containerized Noobaa using IAM API ([AWS documentation](https://docs.aws.amazon.com/iam/)).  
+A created user will be able to get access to NooBaa resources (buckets, objects).
+
+## Background
+- Currently, NooBaa account creation APIs are proprietary and do not adhere to industry standards, necessitating this change.
+  IAM APIs will provide industry standards account APIs, and that will enable customers to create wrapper around it. User can make use of existing scripts that already working for AWS IAM. Customers trying to create real multi-tenant environment, mainly for data scientists, but not only. And also providing the right access control for users.
+
+    ```bash
+    nb account create {account_name} --show-secrets
+    ```
+## Problem
+Limitation of current Noobaa account API
```

Comment:
You need to replace the "Noobaa" with "NooBaa" in the doc.
```suggestion
Limitation of current NooBaa account API
```

Reply:
why?